OneBusAway/otpkit
# Make Origin Destination Working Marker View

# Make Origin Destination Working Marker View

## Related Issues
#22 

## Description
This PR scope is adding marker on origin and destination map

## Media
https://github.com/user-attachments/assets/61e88061-a2bf-4f82-9234-918bacb7ac73

## Test Instructions
1. Try to select select map based on origin or destination. See if the marker shown and stayed there
